src/katsdpcontroller/product_config.py

```diff
@@ -219,18 +219,41 @@ def from_config(cls, config: Mapping[str, Any]) -> "ServiceOverride":
         )
 
 
+class DevelopOptions:
+    def __init__(
+        self, *, any_gpu: bool = True, disable_ibv: bool = True, less_resources: bool = True
+    ) -> None:
+        self.any_gpu = any_gpu
+        self.disable_ibv = disable_ibv
+        self.less_resources = less_resources
+
+    @classmethod
+    def from_config(cls, config: Mapping[str, bool]) -> "DevelopOptions":
+        return cls(
+            any_gpu=config.get("any_gpu", True),
+            disable_ibv=config.get("disable_ibv", True),
+            less_resources=config.get("less_resources", True),
+        )
+
+
 class Options:
     def __init__(
         self,
         *,
         develop: bool = False,
+        develop_opts: Mapping[str, bool] = {},
         wrapper: Optional[str] = None,
         image_tag: Optional[str] = None,
         image_overrides: Mapping[str, str] = {},
         service_overrides: Mapping[str, ServiceOverride] = {},
         interface_mode: bool = False,
     ) -> None:
-        self.develop = develop
+        if not develop:
+            self.develop_opts = DevelopOptions(
+                any_gpu=False, disable_ibv=False, less_resources=False
+            )
+        else:
+            self.develop_opts = DevelopOptions.from_config(develop_opts)
```

Comment:
What do you think about swapping the two sides of the `if`, so that it's `if develop` rather than `if not develop`.

Reply:
I've modified this so I don't think this comment applies anymore